package.json

```diff
@@ -35,22 +35,22 @@
     "@babel/plugin-transform-modules-commonjs": "^7.12.1",
     "@glennsl/bs-jest": "^0.6.0",
     "@reasonml-community/graphql-ppx": "^1.0.1",
+    "@rescript/react": "^0.10.1",
     "all-contributors-cli": "^6.19.0",
     "babel-jest": "^26.6.3",
-    "bs-platform": "^8.3.2",
-    "gentype": "^3.38.0",
+    "bs-platform": "^9.0.1",
+    "gentype": "^3.44.0",
     "graphql": "^15.4.0",
     "npm-run-all": "^4.1.5",
     "react": "^16.8.0",
-    "reason-react": "^0.9.1",
-    "urql": "~1.11.2"
+    "urql": "~2.0.1"
   },
   "peerDependencies": {
     "@reasonml-community/graphql-ppx": "^1.0.1",
+    "@rescript/react": "^0.10.1",
     "graphql": "^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0",
     "react": "^16.8.0",
-    "reason-react": "^0.7.0",
-    "urql": "~1.11.0"
+    "urql": "~2.0.0"
   },
   "resolutions": {
     "wonka": "5.0.0-rc.1"
```

Comment:
I suppose, this is still a permanent fix that all users need to add, right? Let's chat about how we can cleanly migrate all of urql to a new version, since I kind of got stuck on updating everything

Reply:
Sure thing — yeah the internals of how ReScript compiles variants really :rekt: us. I'll see if I can snag some of your time later in March (on PTO next week) to discuss what an upgrade strategy might look like.